src/openseadragon.js

```diff
@@ -1403,6 +1405,26 @@ function OpenSeadragon( options ){
             CHROMEEDGE: 7
         },
 
+        /**
+         * Keep track of which {@link Viewer}s have been created.
+         * - Key: {@link Element} to which a Viewer is attached.
+         * - Value: {@link Viewer} of the element defined by the key.
+         * @static
+         * @type {Object}
+         */
+        // eslint-disable-next-line no-undef
```

Comment:
Can you explain what this lint error is about?

Reply:
`Map` was being marked as undefined by ESLint, although the implementation was working as expected.

I've removed the comment and put `Map` as a global in the files `.eslintrc.*` files, since it can be accessed globally. I'm not sure why ESLint marks it as undefined

I also changed false/true in globals according to this commit message: 94f269d

Reply:
Interesting... probably because we have our language settings for an old version of JS because of our IE support. Anyway, adding Map to the globals seems like a good solution.